src/components/EmptyList/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+function EmptyList() {
+  const history = useHistory();
+
+  const goToAddItemView = () => {
+    history.push('/add-item');
+  };
+
+  return (
+    <section>
+      <p>Your shopping list is currently empty.</p>
+      <button onClick={goToAddItemView}>Add Item</button>
+    </section>
+  );
+}
+
+export default EmptyList;
```

Comment:
Good job, @alxmcr and @macosta08! Just a question, it is a small detail, why was it decided that for this file was `.js` and not `jsx` as so far have been created the other components?
It is a doubt, but this does not prevent the app to work, the code is very organized and clear. Congratulations!

Reply:
Thanks, @Lukaina for your observation and great words! 👍 

Well, I still don't get used to using JSX files but I already changed it. 😄

Reply:
@alxmcr, Many thanks to you for the work and the clarification! Don't worry, it usually happens, it just made me curious. 😄

Reply:
You can read more in the React documentation: https://reactjs.org/docs/introducing-jsx.html 
A way I like to think about it is when a component has any type of HTML inside of it, it is a jsx file. If it is pure javascript, for example, the helpers file, which contain only javascript functions, then it is a .js file. 
Other common js files is when you are using redux, the actions, reducers, and containers will be js files since they don't have any HTML in them.